overwatch/processing/trending/manager.py

```diff
@@ -56,11 +72,27 @@ def _prepareDataBase(objName, dbPosition):  # type: (str, Persistent)-> None
             dbPosition[objName] = BTree()
 
     def createTrendingObjects(self):
-        """ADD DOC"""
+        """ It loops over subsystems and calls function that creates trending objects for each subsystem.
+
+        Args:
+            None.
+        Return:
+            None.
+        """
         for subsystem in self.parameters[CON.SUBSYSTEMS]:
             self._createTrendingObjectsForSubsystem(subsystem)
 
     def _createTrendingObjectsForSubsystem(self, subsystemName):  # type: (str) -> None
+        """  Receives information about trending object and calls function that creates it.
+
+        It imports 'pluginManager' module and for each subsystem tries to get information about trending
```

Comment:
I think that "imports" isn't appropriate word, especially for python, where it is keyword and this function don't perform import operation